index.html

```diff
@@ -229,23 +229,29 @@ <h2 class="primary-header">About me</h2>
     <section class="contact-form">
       <h2 id="contact-header-message"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
       <form action="https://formspree.io/f/xgeqyrjy" method="post">
+        <div id="first-last-name">
+          <input class="field" name="firstName" id="firstName" type="text" placeholder="First Name" maxlength="30" autocomplete="off" required>
+          <input class="field" name="lastName" id="lastName" type="text" placeholder="Last Name" maxlength="30" autocomplete="off" required>
+        </div>
```

Comment:
What a great job. One of the best projects I have seen. :clap: :clap:  However, there is just one tiny issue. Kindly make sure that your contact form in the desktop version is still working. :pray: :pray:  It's not necessary to use two inputs for the name and last name It's ok if you just use one as the mobile version. This is mentioned in the requirements of the project. When you apply this change the contact form stops working.  
![image](https://user-images.githubusercontent.com/98363075/194458675-66aa6291-c53f-4315-8e98-892a0b6eb876.png)
![image](https://user-images.githubusercontent.com/98363075/194459013-c9b5d49c-7d63-49a7-bf37-6e6d3e7ba58f.png)

Reply:
Ok. Great! Thanks!